fms/models/hf/README.md

```diff
@@ -0,0 +1,179 @@
+# Huggingface Adapters in FMS
+
+## What is it?
+
+`HFModelArchitecture` is a set of base classes which provide huggingface functionality to pytorch-native modules. The  
+benefits
+you get from using this method are as follows:
+
+- more flexibility of implementation of your underlying pytorch native classes
+- seamless conversions from pytorch native modules to those using a huggingface API (no longer need to write long weight  
```

Comment:
I tend to prefer 80 but some of these lines are still longer than 120. Also, it would be better to remove the trailing whitespace. The trailing whitespace chars are causing these linebreaks to become hard line breaks in the rendered markdown. E.g. see here when resizing the window it causes these lines to actually separate:
https://github.com/foundation-model-stack/foundation-model-stack/blob/a67ac241c371d38b3f46fbda7afd856a24355795/fms/models/hf/README.md

Reply:
Capped at 80, let me know how it looks now